ckan/lib/repoze_plugins/auth_tkt.py

```diff
@@ -69,9 +92,15 @@ def make_plugin(secret=None,
     if timeout is not None and reissue_time is None:
         reissue_time = int(math.ceil(int(timeout) * 0.1))
     # Set httponly based on config value. Default is True
-    httponly = config.get(u'who.httponly', True)
+    httponly = _bool(config.get(u'who.httponly', True))
```

Comment:
This should probably be deferred until after we check the attributes that might cause an early termination.

Reply:
Boolean conversions were done in CkanAuthTktCookiePlugin call and those variables weren't used in anywhere else, so it makes sense to convert them right away.